src/components/create-exam/step1.tsx

```
@@ -350,33 +343,59 @@ export const Step1 = ({ onNext }: Step1Props) => {
           ☺︎
         </Button>
         <CardHeaderContent>
-          <CardTitle>Let’s create your questions!</CardTitle>
-          <CardDescription>
-            Create questions that inspire, challenge, and engage participants for a truly
-            interactive experience.
-          </CardDescription>
+          <CardTitle>Let's create your questions!</CardTitle>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)